index.html

```diff
@@ -1094,7 +1094,7 @@ <h2 id='get-presences'>Get Presences</h2>
 </tr>
 </tbody></table>
 <h2 id='get-status'>Get Status</h2>
-<p>This function returns the blurb of a user. The blurb is the equivalent of the user&#39;s description on Roblox, not their status message.</p>
+<p>This function returns the status of a user. The blurb is the equivalent of the user&#39;s description on Roblox, not their status message.</p>
```

Comment:
Could you please also replace the other instance of "blurb" with status?
Are there any other improvements you could make, to make this PR even better?